src/modules/shared/services/innovations.service.ts

```diff
@@ -81,6 +81,15 @@ export type GetThreadMessageInfoDTO = {
   createdAt: DateISOType;
 };
 
+export type GetThreadParticipantsDTO = {
+  participants: {
+    id: string;
+    name: string;
+    role: UserTypeEnum;
+    organisationUnit?: { id: string, acronym: string} | undefined
```

Comment:
@Miguel-Ferro-bjss I am not sure but I think using `?:` and `| undefined` is the same thing (double check if you can)